tabbycat/tournaments/forms.py

```diff
@@ -92,6 +92,11 @@ class Meta:
         help_text=_("Show non-sensitive information on the public-facing side of this site, "
             "like draws (once released) and the motions of previous rounds"))
 
+    private_urls = ChoiceField(
+        choices=private_url_presets_for_form(), # Tuple with (Present_Index, Preset_Name)
+        label=_("Use Private URLs"),
+        help_text=_("Enables participant data entry through private URLs."))
+
```

Comment:
I think we should generalize this field, to answer the question "How will submissions be made for the tournament" for which there are 3 answers which would form the options:
* Tab team only
* Private URLs
* Public forms

Tab team only is the default, so would be the default, and `False` as the preset "key". Then, private URLs which you have implemented. Public forms should have a preset as well to complete these options. I'd rename this field and its attributes...

BTW, typo in the comment and it's not really true anyways, as the tuple is (preset name, name for the form)

Reply:
This makes sense, I was sticking with the preset, but instead I could add another preset for the public forms, and then implement it as three choices. The comment was just carried over from the public_info code which I duplicated, happy to remove this when resolving the first comment

Reply:
Additional setting has been implemented and the names have been changed